### PR TITLE
Use Ubuntu 18.04 in Dockerfile

### PR DESCRIPTION
**Pull request by josephfrazier**
This fixes recognition of plates in EXIF-oriented images. See here for
details: https://github.com/openalpr/cloudapi/issues/16#issuecomment-396740100

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from ubuntu:18.04
 
 # Install prerequisites
+env DEBIAN_FRONTEND=noninteractive
 run apt-get update && apt-get install -y \
```
Comment (Silex) [inline Dockerfile]: Don't use `env` for that, use `run DEBIAN_FRONTEND=noninteractive apt-get update...`

See https://github.com/moby/moby/issues/4032

Comment (josephfrazier) [reply Dockerfile]: Oh, I didn't think about that, thanks! Fixed in https://github.com/openalpr/openalpr/pull/690/commits/61e98b4c8b7010a9f5e5ab82d18e7222755cbd84

**Comment by matthill** [comment]
Thanks  Joseph, and Philippe.  I think for the Cloud API we're using Python PIL to read the images.  Hopefully switching to OpenCV 3.1 will resolve it too.

**Comment by josephfrazier** [comment]
Thanks for the merge, @matthill. I hope it's not too difficult to fix the Cloud API (is the code for that online anywhere? I'd be happy to take a look if so)

**Comment by pide2000** [comment]
For me the Docker image on ubuntu 18.04 doens't work very well.  The results are bad or there is simply no result. Test image is ok but every other image doesn't work. So I changed the version to Ubuntu 16.04 and it works like a charm. 
On the 18.04 machine I get in most cases no result, not even an error or "No license plates found." On a local 18.04 machine I get error messages. I think in Docker there are error messsages too, but they are somewhow not passed to the host system.
What do you think?

**Comment by josephfrazier** [comment]
(for reference, the problem described above got it's own issue at https://github.com/openalpr/openalpr/issues/706)

**Pull request merged**